src/screens/AdminStudy.js

```diff
@@ -1,28 +1,53 @@
-import logo from "../logo.svg";
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css'
```

Comment:
This breaks the compilation, but it shouldn't be necessary anyway. If we _need_ the CSS here, consider putting it in `App.css` and adding back in the 
```
// Please use Tailwind CSS (https://tailwindcss.com/)
// instead of plain CSS where possible.
import "../App.css"
```